web3.js/src/connection.js

```diff
@@ -2933,15 +2933,10 @@ export class Connection {
     const skipPreflight = options && options.skipPreflight;
     const preflightCommitment = options && options.preflightCommitment;
 
-    if (skipPreflight && preflightCommitment) {
-      throw new Error(
-        'cannot set preflightCommitment when skipPreflight is enabled',
```

Comment:
@jstarry - I removed this code because it felt to opinionated to me (and it blew up on me).   I prefer the JSON RPC server to make the call about whether this is a problem or not (it currently doesn't care)